HarpGuru/HarpFace/helpers/getHarpFaceFacts/index.ts

```diff
@@ -4,15 +4,14 @@ import { transposeMatrix } from '../transposeMatrix'
 import { getOctaveColumnGroups } from '../getOctaveColumnGroups'
 import type { ColumnRanges } from '../getOctaveColumnGroups'
 import { arrayHasRoot } from '../arrayHasRoot'
-import type { HarpFaceProps } from '../../types'
 
 type HarpFaceFacts = {
   readonly rowCount: number
   readonly columnCount: number
   readonly octaveColumnGroups: ColumnRanges
 }
 
-type Props = HarpFaceProps & {
+type Props = {
```

Comment:
This props needn't be defined now that the params are so simple.

Reply:
Referenced in open issue: https://github.com/js-jslog/harpnative/issues/45#issuecomment-663618566